index.js

```diff
@@ -31,7 +35,13 @@ app.get('/recent', function (req, res) {
   });
 })
 
+app.get('/recent-with-versions', function (req, res) {
```

Comment:
maybe just call this /recents so when we transition over it seems like a less weird name?